web/templates/confirm_dialog/confirm_disable_all_notifications.hbs

```diff
@@ -0,0 +1,12 @@
+<p>
+    {{#tr}}
+    You are about to disable all notifications for private messages,
+    @&#8209;mentions and alerts, which may cause you to miss messages
+    that require your timely attention.
+    If you would like to temporarily disable all desktop notifications,
+    consider <z-link>turning on "Do not disturb"</z-link> instead.
+    {{#*inline "z-link"}}<a href="/help/do-not-disturb">{{> @partial-block}}</a>{{/inline}}
+    {{/tr}}
```

Comment:
I fixed the line-wrapping here and also fixed the URL to be relative; there's no need to hardcode zulip.com just to link to the help center (and it's important that we not do so for self-hosted Zulip servers not connected to the main Internet!).